Makefile

```diff
@@ -1,8 +1,10 @@
 .PHONY: build generate-completion test test-cover view-cover
 
 TAG=$(shell git describe --tags 2> /dev/null || echo unknown)
```

Comment:
Could you remove the `TAG` varible.

Reply:
Fixed at https://github.com/hazelcast/hazelcast-commandline-client/pull/107/commits/4d6e7149dd84650c120e2cd2d9323b80087d1ad9.